larray/core/array.py

```diff
@@ -3108,18 +3110,24 @@ def labelsofsorted(self, axis=None, kind='quicksort'):
              BE  M  F
              FR  F  M
              IT  M  F
+        >>> arr.labelsofsorted(X.sex, reverse=True)
+        nat\\sex  1  0
```

Comment:
I don't think this is the result we want. At least that was not what I needed when I wanted reverse: I expected:
```python
         nat\\sex  0  1
              BE  F  M
              FR  M  F
              IT  F  M
```
The reasoning is this: I want to sort from highest to lowest, and given that sort, I want to know which is the first element (ie indice 0), which is second(1), etc. Not just display them in flipped order.